test/transactions/put_block_blob_and_snapshot/0_response.json

```diff
@@ -0,0 +1 @@
+{"status":200,"headers":{"content-type":"application/xml","date":"Wed, 03 Aug 2022 14:22:13 GMT","server":"Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0","transfer-encoding":"chunked","x-ms-request-id":"bf2dc2eb-601e-0065-2044-a7c5f3000000","x-ms-version":"2019-12-12"},"body":"77u/PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48RW51bWVyYXRpb25SZXN1bHRzIFNlcnZpY2VFbmRwb2ludD0iaHR0cHM6Ly9ibGFja2JpcmRpbmRleGRldi5ibG9iLmNvcmUud2luZG93cy5uZXQvIj48Q29udGFpbmVycz48Q29udGFpbmVyPjxOYW1lPmNpLWJ1aWxkczwvTmFtZT48UHJvcGVydGllcz48TGFzdC1Nb2RpZmllZD5UaHUsIDE0IEp1bCAyMDIyIDIyOjQzOjUxIEdNVDwvTGFzdC1Nb2RpZmllZD48RXRhZz4iMHg4REE2NUVBNTMzRjY3RkIiPC9FdGFnPjxMZWFzZVN0YXR1cz51bmxvY2tlZDwvTGVhc2VTdGF0dXM+PExlYXNlU3RhdGU+YXZhaWxhYmxlPC9MZWFzZVN0YXRlPjxEZWZhdWx0RW5jcnlwdGlvblNjb3BlPiRhY2NvdW50LWVuY3J5cHRpb24ta2V5PC9EZWZhdWx0RW5jcnlwdGlvblNjb3BlPjxEZW55RW5jcnlwdGlvblNjb3BlT3ZlcnJpZGU+ZmFsc2U8L0RlbnlFbmNyeXB0aW9uU2NvcGVPdmVycmlkZT48SGFzSW1tdXRhYmlsaXR5UG9saWN5PmZhbHNlPC9IYXNJbW11dGFiaWxpdHlQb2xpY3k+PEhhc0xlZ2FsSG9sZD5mYWxzZTwvSGFzTGVnYWxIb2xkPjwvUHJvcGVydGllcz48L0NvbnRhaW5lcj48Q29udGFpbmVyPjxOYW1lPmRldmVsb3BtZW50PC9OYW1lPjxQcm9wZXJ0aWVzPjxMYXN0LU1vZGlmaWVkPlRodSwgMTQgSnVsIDIwMjIgMjI6NDM6NTIgR01UPC9MYXN0LU1vZGlmaWVkPjxFdGFnPiIweDhEQTY1RUE1MzQ0NkMzQiI8L0V0YWc+PExlYXNlU3RhdHVzPnVubG9ja2VkPC9MZWFzZVN0YXR1cz48TGVhc2VTdGF0ZT5hdmFpbGFibGU8L0xlYXNlU3RhdGU+PERlZmF1bHRFbmNyeXB0aW9uU2NvcGU+JGFjY291bnQtZW5jcnlwdGlvbi1rZXk8L0RlZmF1bHRFbmNyeXB0aW9uU2NvcGU+PERlbnlFbmNyeXB0aW9uU2NvcGVPdmVycmlkZT5mYWxzZTwvRGVueUVuY3J5cHRpb25TY29wZU92ZXJyaWRlPjxIYXNJbW11dGFiaWxpdHlQb2xpY3k+ZmFsc2U8L0hhc0ltbXV0YWJpbGl0eVBvbGljeT48SGFzTGVnYWxIb2xkPmZhbHNlPC9IYXNMZWdhbEhvbGQ+PC9Qcm9wZXJ0aWVzPjwvQ29udGFpbmVyPjxDb250YWluZXI+PE5hbWU+Z29yemVsbDwvTmFtZT48UHJvcGVydGllcz48TGFzdC1Nb2RpZmllZD5UaHUsIDE0IEp1bCAyMDIyIDIyOjQzOjUxIEdNVDwvTGFzdC1Nb2RpZmllZD48RXRhZz4iMHg4REE2NUVBNTMzQkVFRjgiPC9FdGFnPjxMZWFzZVN0YXR1cz51bmxvY2tlZDwvTGVhc2VTdGF0dXM+PExlYXNlU3RhdGU+YXZhaWxhYmxlPC9MZWFzZVN0YXRlPjxEZWZhdWx0RW5jcnlwdGlvblNjb3BlPiRhY2NvdW50LWVuY3J5cHRpb24ta2V5PC9EZWZhdWx0RW5jcnlwdGlvblNjb3BlPjxEZW55RW5jcnlwdGlvblNjb3BlT3ZlcnJpZGU+ZmFsc2U8L0RlbnlFbmNyeXB0aW9uU2NvcGVPdmVycmlkZT48SGFzSW1tdXRhYmlsaXR5UG9saWN5PmZhbHNlPC9IYXNJbW11dGFiaWxpdHlQb2xpY3k+PEhhc0xlZ2FsSG9sZD5mYWxzZTwvSGFzTGVnYWxIb2xkPjwvUHJvcGVydGllcz48L0NvbnRhaW5lcj48Q29udGFpbmVyPjxOYW1lPnRlc3Q8L05hbWU+PFByb3BlcnRpZXM+PExhc3QtTW9kaWZpZWQ+VGh1LCAxNCBKdWwgMjAyMiAyMjo0Mzo1MSBHTVQ8L0xhc3QtTW9kaWZpZWQ+PEV0YWc+IjB4OERBNjVFQTUzNDE2OTI3IjwvRXRhZz48TGVhc2VTdGF0dXM+dW5sb2NrZWQ8L0xlYXNlU3RhdHVzPjxMZWFzZVN0YXRlPmF2YWlsYWJsZTwvTGVhc2VTdGF0ZT48RGVmYXVsdEVuY3J5cHRpb25TY29wZT4kYWNjb3VudC1lbmNyeXB0aW9uLWtleTwvRGVmYXVsdEVuY3J5cHRpb25TY29wZT48RGVueUVuY3J5cHRpb25TY29wZU92ZXJyaWRlPmZhbHNlPC9EZW55RW5jcnlwdGlvblNjb3BlT3ZlcnJpZGU+PEhhc0ltbXV0YWJpbGl0eVBvbGljeT5mYWxzZTwvSGFzSW1tdXRhYmlsaXR5UG9saWN5PjxIYXNMZWdhbEhvbGQ+ZmFsc2U8L0hhc0xlZ2FsSG9sZD48L1Byb3BlcnRpZXM+PC9Db250YWluZXI+PC9Db250YWluZXJzPjxOZXh0TWFya2VyIC8+PC9FbnVtZXJhdGlvblJlc3VsdHM+"}
```

Comment:
I think you need to run `./eng/scripts/format_json.sh` so that all json is formatted.